### DISCUSSION TITLE
[question]请问紧凑树和生态树有什么区别？

### DISCUSSION THREAD

**Discussion by liaodalin19903** [antvis/G6#6594] [Q&A] [answered]
在G6官网文档有看到紧凑树和生态树2种布局类型类型：未见差异。
![image](https://github.com/user-attachments/assets/c184cb0a-c2ae-4da5-98cc-17ecad15f34e)

请问它们之间有什么不同之处呢？

**Comment by yvonneyx** [comment, marked as answer]
- 生态树：不管数据的深度多少，总是叶节点对齐
- 紧凑树：从根节点开始，同一深度的节点在同一层
